files/SZWESAMP/ZWEITSS

```diff
@@ -215,9 +215,16 @@ TSS PERMIT({zowe.setup.security.users.zowe}) APPL(OMVSAPPL)
 
 /* DEFINE ZOWE RESOURCE PROTECTION ................................. */
 
+/* - Defines new resource class for Zowe that protects access to     */
+/*   sensitive Zowe resources.                                       */
 /* - Defines resource APIML.SERVICES that controls access to         */
 /*   detailed information about API services to Zowe users.          */
 
+/* define ZOWE resource class                                        */
+/* skip this command if the ZOWE resource class already exists       */
```

Comment:
I asked one of the Rocket admins if this is needed on RACF/TSS/ACF2.
I know its not needed for RACF since zos 2.5 (and so this PR was updated, thanks)
But the admin also sees it present on TSS, but not on ACF2.
So it could be that this block is only needed for ACF2???

They were going to ask Broadcom for more info but, if anyone knows for sure when/if TSS/ACF2 versions have already included the class, it would be good to capture this knowledge. Otherwise, we could be producing an error when running this job?

Reply:
OK - they heard back that indeed ACF2 does not include this class.
This leaves uncertainty about TSS. Who can confirm if TSS really needs the class defined by us rather than that it ships it?

Reply:
I have now heard back that TSS DOES include this class.
So, only ACF2 needs this change.